cni/pkg/iptables/iptables.go

```diff
@@ -616,7 +617,6 @@ func (cfg *IptablesConfigurator) DeleteHostRules() {
 }
 
 func (cfg *IptablesConfigurator) AppendHostRules() *builder.IptablesRuleBuilder {
-	log.Info("configuring host-level iptables rules (healthchecks, etc)")
```

Comment:
unrelated to the issue:
During testing, I noticed that we were printing this log line twice since it is used to create the IptablesRuleBuilder for both the creation and deletion of the rules. I moved it to only print during creation to ensure consistency with the in-pod rules handling.